openapi/components/schemas/BalanceTransactions/SellFeeTransaction.yaml

```
@@ -26,8 +26,10 @@ properties:
 
       Some types of transactions relate to parent transactions.
       For example, a `risk-reserve` relates to a `charge`, and a `risk-reserve-release` relates a to `risk-reserve`.
-    allOf:
-      - $ref: ../ResourceId.yaml
+    type: string
+    nullable: true
+    maxLength: 50
+    example: 4f6cf35x-2c4y-483z-a0a9-158621f77a21
```

Since it's a balance transaction parent id, I think we should specify the same id ULID format as above.

```suggestion
    example: btxn_0YVCNJYRXTDDQ9QF2Z60ZA05WP
```